gnucash/gnome-utils/gnc-account-sel.h

```diff
@@ -38,24 +38,29 @@
 #define GNC_ACCOUNT_SEL_CLASS(klass)  G_TYPE_CHECK_CLASS_CAST (klass, GNC_TYPE_ACCOUNT_SEL, GNCAccountSelClass)
 #define GNC_IS_ACCOUNT_SEL(obj)       G_TYPE_CHECK_INSTANCE_TYPE (obj, GNC_TYPE_ACCOUNT_SEL)
 
+#define BUFLEN 1024
```

Comment:
Generic defines at global scope like this are a bad idea. Give it a name that's unlikely to be used elsewhere, perhaps `GNCACCTSEL_PREFIX_BUFLEN`.

Better still, put the struct in the implementation file and only expose the typedef in the header:
`typedef GncAccountSel struct _GncAccountSel`

Reply:
OK, I thought I had done this in one of my changes locally, done now.